dom0/securedrop-update

```diff
@@ -21,26 +21,34 @@ function securedrop-update-feedback() {
     shift
 
     # Running `notify-send` as root doesn't work, must be normal user.
-    # Setting 30s expire time (in ms) since it's a long-running cmd.
-    su user -c "notify-send \
+    # Setting 60s expire time (in ms) since it's a long-running cmd.
+    local qubes_user
+    qubes_user="$(id -nu 1000)"
+    su "$qubes_user" -c "notify-send \
+        --app-name 'SecureDrop Workstation' \
         --icon /usr/share/securedrop/icons/sd-logo.png \
-        --expire-time 30000 \
-        '$msg'"
+        --expire-time 60000 \
+        'SecureDrop: $msg'"
 }
 
 # `qubesctl pkg.upgrade` will automatically update dom0 packages, as well,
 # but we *first* want the freshest RPMs from dom0, *then* we'll want to
 # update the VMs themselves.
-securedrop-update-feedback "SecureDrop: Updating dom0 configuration..."
-sudo qubes-dom0-update -y
 
-securedrop-update-feedback "SecureDrop: Updating application..."
-qubesctl --templates \
+securedrop-update-feedback "Updating application..."
+qubesctl \
```

Comment:
Oh, sorry, I think I wasn't clear. My previous comment was about the ordering only, not which actions are performed. If you want to update templates, you still need `--templates`. If you want to apply configuration to other vms (non-templates), then you need `--all`.

Reply:
That would explain the behavior I'm seeing locally, @marmarek; many thanks for your guidance here!